bukkit/src/main/java/io/github/wysohn/triggerreactor/bukkit/bridge/AbstractBukkitWrapper.java

```diff
@@ -28,6 +26,8 @@ public IEntity wrap(Entity entity) {
 
     public abstract IPlayer wrap(Player player);
 
+    public abstract IConsoleCommandSender wrap(ConsoleCommandSender sender);
```

Comment:
Is implementation for BukkitConsoleCommandSender differ between latest and legacy versions? Or we could just make it a concrete method